versions.gradle

```diff
@@ -24,7 +24,7 @@ ext.deps = [:]
 def versions = [:]
 // GeckoView versions can be found here:
 // https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/
-versions.gecko_view = "68.0.20190422094240"
+versions.gecko_view = "68.0.20190506130308"
```

Comment:
since it's been a few days, I bumped the version: `68.0.20190508111321`. seems to work well so far.